src/components/Filters/FilterDivider.tsx

```diff
@@ -0,0 +1,12 @@
+/**
+ * A component that displays numerical facets applicable to the current vertical search.
+ *
+ * @param props - A customizable className for facet divider
+ * @returns A React component for facets
+ *
+ * @public
+ */
+
+export function FilterDivider({ className ='w-full h-px bg-gray-200 my-4' }: { className?: string }): JSX.Element {
```

Comment:
should we be using `tw-merge` for the `className` to allow deep merging? it might be a little confusing if users specify it with the `customCssClasses` of a component like `StandardFacets` where the other classes would be merged, but this one would be replaced